nitro-root-enclave/src/main.rs

```diff
@@ -82,12 +81,18 @@ static AWS_NITRO_ROOT_CERTIFICATE: [u8; 533] = [
 const NSM_MAX_ATTESTATION_DOC_SIZE: usize = 16 * 1024;
 
 lazy_static! {
-    /// The (randomly) self-generated device key pair
-    static ref DEVICE_KEY_PAIR: Mutex<Option<(EcdsaKeyPair, Vec<u8>)>> = Mutex::new(None);
+    /// The (randomly) self-generated device key pair.
+    /// Would like to have this stored as EcdsaKeyPair, instead of having
+    /// to generate it from bytes each time, but EcdsaKeyPair doesn't support
+    /// `copy` or `clone`, so it's hard to manage that way
+    static ref DEVICE_KEY_PAIR: Mutex<Option<Vec<u8>>> = Mutex::new(None);
     /// The hash value of the Runtime Manager enclave
     static ref RUNTIME_MANAGER_HASH: Mutex<Option<Vec<u8>>> = Mutex::new(None);
     /// The Device ID assigned to us by the Proxy Attestation Service
     static ref DEVICE_ID: Mutex<Option<i32>> = std::sync::Mutex::new(None);
+    static ref CERT_CHAIN: Mutex<Option<(Vec<u8>, Vec<u8>)>> = Mutex::new(None);
+    static ref CHALLENGE_ID_COUNTER: AtomicI32 = AtomicI32::new(0);
```

Comment:
Is this more naturally an `AtomicU32`?  That's the type used in `trustzone-root-enclave` for instance...